src/lib/features/events/event-service.ts

```diff
@@ -222,6 +229,14 @@ export default class EventService {
     async getEventCreators() {
         return this.eventStore.getEventCreators();
     }
+
+    async getProjectFilterForNonAdmins(userId: number): Promise<IQueryParam[]> {
```

Comment:
Small suggestion here: the naming was a little unclear to me at first, because it doesn't seem like we know whether the user is an admin or not at this point. If you read the method, you'll see that it's checked in there, though.

I'm not sure I have any better suggestions, but maybe `getRoleBasedFilters` would work? I ... dunno 🤷🏼